packages/apps/human-app/frontend/src/shared/components/ui/page-card/page-card.tsx

```diff
@@ -197,7 +168,7 @@ export function PageCard({
               >
                 <ArrowBackIcon fontSize="inherit" />
               </IconWrapper>
-            ) : null}
+            )}
             {!hiddenCancelButton && (
               <Button
                 onClick={() => {
```

Comment:
This and others anonymous function can be moved to some `handle<btn>Click` (e.g. `handleCancelButtonClick`) for better readability: ideally JSX in `return` should carry only view-related code, so you don't care about business part and can think just of UI